RevenueCatUI/CustomerCenter/Views/NoSubscriptionsView.swift

```diff
@@ -70,17 +71,31 @@ struct NoSubscriptionsView: View {
                     showRestoreAlert = true
                 }
                 .restorePurchasesAlert(isPresented: $showRestoreAlert)
-                .buttonStyle(ManageSubscriptionsButtonStyle())
-
-                Button(localization.commonLocalizedString(for: .cancel)) {
-                    dismiss()
-                }
-                .padding(.vertical)
+                .buttonStyle(ProminentButtonStyle())
             }
             .padding(.horizontal)
             .applyIf(textColor != nil, apply: { $0.foregroundColor(textColor) })
         }
+        .toolbar {
+            ToolbarItem(placement: .topBarTrailing) {
+                DismissCircleButton {
+                    dismiss()
+                }
+            }
+        }
+
+    }
 
+    var body: some View {
+        if #available(iOS 16.0, *) {
+            NavigationStack {
+                content
+            }
+        } else {
+            NavigationView {
+                content
+            }
+        }
     }
```

Comment:
This view will _always_ be presented modally in some manner? If so this is fine, but if someone were to add this to an existing navigation stack, weird things could happen.

Additionally, I see we've used this pattern twice, if we use it a third time, should consider creating a component we can reuse for this.

Reply:
The modal question is one we'll need to answer, good call 👍 

Agreed on the reusable component - will pull this out into a new component in a separate PR.